src/models/customerModel.ts

```diff
@@ -0,0 +1,27 @@
+
+import dbConnection from "../dbConnection/dbConnection";
+
+export interface User {
```

Comment:
Again, I think it would be better for all interface names to start with an uppercase I, so this would be `IUser`